src/conv.rs

```diff
@@ -179,6 +179,17 @@ pub fn map_origin3d(native: &native::WGPUOrigin3D) -> wgt::Origin3d {
     }
 }
 
+pub fn map_adapter_options<'a>(
+    options: &native::WGPURequestAdapterOptions,
+    extras: Option<&native::WGPUAdapterExtras>,
+) -> (Option<id::SurfaceId>, i32) {
+    if let Some(extras) = extras {
+        (Some(options.compatibleSurface), extras.backendBits)
```

Comment:
I'm not so sure about the `Some` here. The `compatibleSurface` can be null, right?